src/services/notification/index.js

```diff
@@ -1,7 +1,7 @@
 const request = require('request-promise-native');
 const hooks = require('./hooks/index');
 
-const { REQUEST_TIMEOUT } = require('../../../config/globals');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `./hooks/index`](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)